petastorm/arrow_reader_worker.py

```diff
@@ -61,10 +61,9 @@ def read_next(self, workers_pool, schema, ngram):
                     try:
                         result_dict[column.name] = np.vstack(list_of_lists.tolist())
                     except ValueError:
-                        raise RuntimeError('Length of all values in column \'%s\' are expected to be the same length. '
-                                           'Got the following set of lengths: \'%s\'',
-                                           column.name,
-                                           ', '.join({value.shape[0] for value in list_of_lists}))
+                        raise RuntimeError("""Length of all values in column '%s' are expected to be the same length.
```

Comment:
Thank you for the fix!
To better align with coding style in the rest of the codebase, can you please:
1. Implement multiline string and string formating like this:
```python
('Length of all values {}...'
 'Got the folling {} ...')
.format(column.name, ', '.join(str(value.shape[0]) for value in list_of_lists)))
```
2. Do you need to construct a set (the curly brackets)? I think an iterable should work fine, i.e.
```python
', '.join({str(value.shape[0]) for value in list_of_lists})
```
-->
```python
', '.join(str(value.shape[0]) for value in list_of_lists)
```

Reply:
Thank you for your comment. 
Added multiline string formatting and removed the set :)